save-backend/src/main/resources/application-dev.properties

```diff
@@ -1,7 +1,12 @@
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
 # use hibernate to populate DB using data.sql
 spring.datasource.initialization-mode=always
-spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.hibernate.ddl-auto=none
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
-spring.jpa.properties.hibernate.validator.apply_to_ddl=false
\ No newline at end of file
+spring.jpa.properties.hibernate.validator.apply_to_ddl=false
+spring.liquibase.change-log=db/changelog/db.changelog-master.xml
+spring.liquibase.contexts=dev
+spring.datasource.url=172.17.0.2:3306
```

Comment:
Please add a comment here that actually in tests we use host that is retrieved from testcontainers, so this IP should be manually changed for manual testing when needed.